### ISSUE TITLE
Support AWS KMS keys

### ISSUE THREAD

**Issue by doctoriggs** [XML-Security/signxml#170]
I have a need to use an asymmetric key held in AWS KMS for signing. I have already started work on this change and I'm happy to submit a PR for it once complete, but I would love some feedback on how best to publish this (I can't be the only one who would use this change). If you think that it should be a separate fork or something that's totally fine. Just trying to contribute my changes back if possible.

**Comment by kislyuk** [comment]
Thanks for opening the issue. I think any functionality that ties directly into the AWS APIs is probably outside the scope of this library; ideally, if you want to share your code, you could publish it as a separate library, or provide a code sample here and I could include it as a submodule with its dependencies (like boto3) installed via a feature flag (`pip install signxml[kms]`).

**Comment by doctoriggs** [comment]
Yeah, I love that answer. Unfortunately, the underlying reasons that I was going to have bandwidth to do this work has changed so we are no longer pursuing this change. I'll close this issue for now and perhaps circle back some time if it gains traction again. Sorry for the churn.
